assets/src/edit-story/components/panels/test/captions.js

```diff
@@ -46,7 +46,7 @@ describe('Panels/Captions', () => {
 
   beforeAll(() => {
     localStorage.setItem(
-      'web_stories_ui_panel_settings:caption',
+      'web_stories_ui_panel_settings:captions',
```

Comment:
Why change this local storage key? Would this reset the stored collapse state of this panel for all users?

Reply:
Got you. You are right, in this case, we should not change the stored state. Changed it back.